FlowCryptUI/Cell Nodes/CellNode.swift

```diff
@@ -9,6 +9,23 @@
 import AsyncDisplayKit
 
 open class CellNode: ASCellNode {
+    var leftBorder: ASDisplayNode?
+
+    func addLeftBorder(width: CGFloat, color: UIColor?) {
```

Comment:
This left border is a nice addition 👍 But let's show it only for encrypted messages, while plain messages won't have this border, so it'll be easier for users to differentiate between secure encrypted messages and plain messages.

Reply:
Ok

Reply:
I have another idea.
I think better to add another border color for unecrypted message? something like lightgray or something like that?

Reply:
Sounds good 👍